src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -387,7 +391,12 @@ intersection(Left, Right, Intersection) ->
 %%
 -spec map(id(), function(), id()) -> ok | error().
 map(Id, Function, AccId) ->
-    gen_server:call(?MODULE, {map, Id, Function, AccId}, infinity).
+    case lasp_config:get(delta_mode, false) of
```

Comment:
Maybe we control this using another variable as well, like to enable or disable incremental computation?  We need a way to evaluate deltas with this enabled and with this disabled.